### PR TITLE
OKTA-96281

### PR DESCRIPTION
**Pull request by ashishgupta-okta**
API changes for remove UD property 

Primary Reviewer: @royalchan-okta  @mystiberry-okta 
Reviewers : @federations-okta

### CODE REVIEW

File: _source/docs/api/resources/schemas.md [hunk 1]
```diff
@@ -814,7 +814,8 @@ curl -v -X POST \
 
 <span class="api-uri-template api-uri-get"><span class="api-label">POST</span> /meta/schemas/user/default</span>
 
-Removes one or more [custom user profile properties](#user-profile-schema-property) from the user schema
+Removes one or more [custom user profile properties](#user-profile-schema-property) from the user schema.
+Property cannot be removed if it is being referenced as a [matchAttribute](./idps.html#subject-policy-object) in SAML2 IdPs.
```
Comment (royalchan-okta) [inline _source/docs/api/resources/schemas.md]: `A property`...

Comment (ashishgupta-okta) [inline _source/docs/api/resources/schemas.md]: changed

**Comment by mystiberry-okta** [comment]
@abinashbehera-okta Looks good--will merge next week assuming OKTA-96281 is part of code freeze this week.

**Comment by mystiberry-okta** [comment]
@ashishgupta-okta Looks great, will merge next week assuming this gets into CF this week.

**Comment by mystiberry-okta** [comment]
@ashishgupta-okta  can you resolve the conflicts so I can merge this? Thanks!

**Pull request merged**